content/docs/Configuration/_index.md

```diff
@@ -769,6 +769,10 @@ jobs:
 
 This config would update Fedora Rawhide and Fedora 35 dist-git branches.
 
+If you need to do any change in the pull request, you need to locally fetch the source branch 
+of the Packit's pull request and push it (with a fix) to your fork (as it is not possible to push to the branch 
+created in the Packit's fork).
+
```

Comment:
I'd definitely show the commands since this is not a trivial git operation for many.
```
git fetch ssh://$USER.fedoraproject.org/forks/packit/rpms/$YOUR_PACKAGE.git refs/heads/*:refs/remotes/packit/*
git cherry-pick packit/$VERSION-$BRANCH-update-propose_downstream
```

I'll pay for a drink if someone can do it with a single command :)